src/main/java/uk/gov/justice/digital/nomis/jpa/entity/CaseloadAgencyIdentity.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.nomis.jpa.entity;
+
+import lombok.*;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.io.Serializable;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@EqualsAndHashCode(of = {"agencyLocationId", "caseload"})
+@Embeddable
+@ToString(of = {"agencyLocationId", "caseload"})
+public class CaseloadAgencyIdentity implements Serializable {
```

Comment:
Isn't this class just modelling a join table? Do we need it, or can we do clevers with the JPA?

Reply:
No, it has extra attributes

Reply:
ok